x/ethbridge/client/cli/tx.go

```diff
@@ -55,3 +55,34 @@ func GetCmdCreateEthBridgeClaim(cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+// GetCmdBurnEth is the CLI command for burning some of your eth and triggering an event
+func GetCmdBurn(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "burn [cosmos-sender-address] [ethereum-receiver-address] [amount]",
+		Short: "burn some coins!",
```

Comment:
this needs a better description. Are these coins burned on the cosmos side or on ethereum? are they transferred to a zero-address or effectively burned (i.e deducted from the total supply)?

Reply:
@musnit I have the same question re: burning. Could you please link here to the code that describes burning?

Reply:
burned cosmos side, and fully burned+deducted (ie, intended to be used for burning cETH or cERC20 representative tokens.

will update description

@ShrutiAppiah https://github.com/cosmos/peggy/pull/79/files#diff-18ba553191438a4534ec11e152406580R92

Reply:
https://github.com/cosmos/peggy/pull/79/commits/b47a8fdf2368e425c08b1751909d5be8dde3e6cf